docs/v8-node.md

```diff
@@ -75,8 +99,11 @@ See [New Performance APIs](./v8-new-performance-apis.md) for details.
 
 ### ESM Support
 
-For now, ESM support is only experimental. For the time being we only fully support CJS-based Node application - we are
-working on this during the v8 alpha/beta cycle.
+Instrumentation works out of the box for CommonJS (CJS) applications based on require() calls. This means that as long
+as your application is either natively in CJS, or compiled at build time to CJS (which is the case for most TypeScript
+apps, for example), everything will work without any further setup.
```

Comment:
```suggestion
as your application is either natively in CJS, or compiled at build time to CJS, everything will work without any further setup.
```

This may not hold over time and may lead people to false security.